frontend/libs/features/calculators/src/components/calculator/calculator.component.ts

```diff
@@ -136,7 +136,14 @@ export class CalculatorComponent implements OnInit, OnChanges, OnDestroy {
 
   pdf(hidePdfComment: boolean) {
     this.hidePdfComment = hidePdfComment;
-    setTimeout(() => window.print(), 50);
+
+    setTimeout(() => {
+      try {
+        document.execCommand('print', false, null!);
```

Comment:
Kun je hier een comment boven zetten dat dit is om safari werkend te krijgen? Ziet er anders beetje appart uit